Makefile

```diff
@@ -91,16 +90,16 @@ MAKE           ?= make
 MAKE_FLAGS     ?= -r
 
 # Version control system
-HG                  ?= hg
-DEFAULT_BRANCH_NAME ?= default
+GIT                 ?= git
+DEFAULT_BRANCH_NAME ?= master
 HG_ARCHIVE_FLAGS    ?= -X .hgtags -X .hgignore -X .devzone -X scripts/make_changelog.sh
 
 # Text processing and scripting
 AWK            ?= awk
 GREP           ?= grep
 PYTHON         ?= python
 UNIX2DOS       ?= $(shell which unix2dos)
-UNIX2DOS_FLAGS ?= $(shell [ -n $(UNIX2DOS) ] && $(UNIX2DOS) -q --version 2>/dev/null && echo "-q" || echo "")
+UNIX2DOS_FLAGS ?= $(shell [ -n $(UNIX2DOS) ] && $(UNIX2DOS) -q --version >/dev/null && echo "-q" || echo "")
```

Comment:
I just ran into the problem that is fixed by this line (unix2dos producing stdout output, that ends up inside `UNIX2DOS_FLAGS`). Good to have it fixed, but I wonder if this fix should be in its own commit?

Regardless, this should probably redirect both stderr and stdout (so `>/dev/null 2>/dev/null`) to prevent issues when unix2dos decides to produce output on stderr too later?

Reply:
Made a new PR out of this: https://github.com/OpenTTD/OpenGFX/pull/11